src/BoE-Info.plist

```diff
@@ -151,21 +151,25 @@
 		</dict>
 	</array>
 	<key>CFBundleExecutable</key>
-	<string>Blades of Exile</string>
+	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIconFile</key>
 	<string>BoE</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.spidweb.bladesofexile</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
This moves the domain to the Xcode project. For whatever reason, Apple does this when upgrading older projects. If this is a problem, it can be reverted.